grails-app/domain/org/broadinstitute/orsp/Issue.groovy

```diff
@@ -205,4 +207,14 @@ class Issue {
         extraproperties
     }
 
+    /** AttachmentApproved verifies if there's any attachment status different to 'Approved'
+     *  if there is any, it will return a value different to null, meaning it still has attachments unReviewed
+     *
+     * @return Boolean indicating if all attachments have 'Approved' status
+     */
```

Comment:
Nit: spacing and language: 
```suggestion
    /** 
     * Determines if there's any status on attachments other than 'Approved', meaning 
     * the issue still has unapproved or unreviewed attachments.
     *
     * @return True if all attachments have the 'Approved' status, false otherwise
     */
```